### ISSUE TITLE
How to generate ground truth pictures like those on paper

### ISSUE THREAD

**Issue by 2000LuoLuo** [MengyangPu/EDTER#101]
Hello,
I want to generate ground truth pictures like those on paper, what should I do?

Below is the execution result of the conversion code I found on the Internet.
![376086](https://github.com/user-attachments/assets/cb57ca12-8969-4fcf-b60b-b7e52e8adfbe)

I want to generate like this.
![2024-10-03T02_43_50](https://github.com/user-attachments/assets/021a234e-4ba1-4747-bd5e-2572771ba518)

Thank You.

**Comment by Gao-yudong** [comment]
change the dataset

**Comment by 2000LuoLuo** [comment]
Yes, I try to converse the image from .mat to png like above, but it doesn't have detail like paper. What should I do? Thanks.

**Comment by Gao-yudong** [comment]
i don't understand it. are you just need the result?

**Comment by LanqingGuo** [comment]
你不可以将结果转为01结果，保留浮点数进行黑白转换
